tutorials/Tutorial_Gallery.ipynb

```diff
@@ -181,10 +181,17 @@
     ]
    },
    "source": [
+    "- [Dimension handling](.Dimension_handling.ipynb)\n",
```

Comment:
```suggestion
    "- [Dimension Handling](.Dimension_handling.ipynb)\n",
```
Would capitalising the "H" in handling (for greater consistency with how the other tutorials on the Tutorial Gallery page) cause any problems? If it will cause problems, please ignore. If it won't cause problems, it would be more consistent.

Reply:
Updated